### ISSUE TITLE
[tvOS] test-tvos-device.log is >2Gb

### ISSUE THREAD

**Issue by MaximLipnin** [dotnet/xharness#731]
For some library test suites  in https://github.com/dotnet/runtime/pull/59503, `test-tvos-device.log` file in CI is > 2Gb.
For example, `System.Net.NetworkInformation.Functional.Tests` ([artifacts](https://dev.azure.com/dnceng/public/_build/results?buildId=1393243&view=ms.vss-test-web.build-test-results-tab&runId=40508598&resultId=155996&paneView=dotnet-dnceng.dnceng-anon-build-release-tasks.helix-anon-test-information-tab))

**Comment by premun** [comment]
Yeah, we already noticed (#691) just didn't have time to look into this
